dummy.env

```diff
@@ -31,4 +31,4 @@ REDIS_PASS=
 GENESIS_URL=http://localhost:9000/genesis
 STORAGE_PATH=/tmp/tails-files
 LOG_LEVEL=INFO
-TAILS_SERVER_URL=http://localhost:6543
+TAILS_URL=http://tails-server:6543
```

Comment:
@mac-arrap this was what I had wrong in that other PR and why it was failing 🤦  On my mac I had it correct, but in circle it uses whatever is in dummy.env . That's a good thing to watch out for when debugging, if things are working locally but failing in ci compare your .env to the dummy.env